frontend/src/components/viewers/__snapshots__/Tensorboard.test.tsx.snap

```diff
@@ -2,34 +2,241 @@
 
 exports[`Tensorboard does not break on empty data 1`] = `
 <div>
-  <BusyButton
-    busy={true}
-    className="buttonAction"
-    onClick={[Function]}
-    title="Start Tensorboard"
-  />
-</div>
-`;
-
-exports[`Tensorboard does not break on empty data 2`] = `
-<div>
-  <BusyButton
-    busy={true}
-    className="buttonAction"
-    onClick={[Function]}
-    title="Start Tensorboard"
-  />
+  <div>
```

Comment:
The snapshot has grown too long, please take a look at which snapshots are useful and delete others.

Recommended set up:
1. snapshot a baseline component
2. use https://github.com/jest-community/snapshot-diff to snapshot difference after certain actions instead of snapshotting the whole component

Further reading: https://kentcdodds.com/blog/effective-snapshot-testing

Reply:
Will modify the snapshots.

Reply:
Thanks! Looks much better now